test/spec/sanity-checks.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+
+var request = require('supertest')
+var assert = require('assert')
+var path = require('path')
+var fs = require('fs')
+
+/**
+ * Basic sanity checks on the dev server
+ */
+describe('The prototype kit', function () {
+  var app
+  var server
+
+  before(function () {
+    app = require('../../server')
+    server = app.listen(3000)
```

Comment:
These appear to be the relevant lines in supertest which listen and close respectively:

https://github.com/visionmedia/supertest/blob/master/lib/test.js#L59

https://github.com/visionmedia/supertest/blob/master/lib/test.js#L126